.aws/src/event-rules/prospect-events/eventConfig.ts

```diff
@@ -1,4 +1,8 @@
 export const eventConfig = {
   source: 'prospect-events',
   detailType: ['prospect-generation'],
+  dismissProspect: {
+    source: 'prospect-events',
+    detailType: ['dismiss-prospect'],
+  },
```

Comment:
i'm a little confused on the organization here - should we refactor the `prospect-generation` event to be formatted like the `dismiss-prospect` event?